src/services/labellang/KeyLabelLangs.ts

```diff
@@ -1,10 +1,118 @@
 import { KeyLabel } from './KeyLabel';
-import { KeyLabelJp } from './KeyLabelJp';
+import { KeyLabelBelgian } from './KeyLabelBelgian';
+import { KeyLabelBepo } from './KeyLabelBepo';
+import { KeyLabelBr_abnt2 } from './KeyLabelBr_abnt2';
+import { KeyLabelCanadian_multilingual } from './KeyLabelCanadian_multilingual';
+import { KeyLabelColemak } from './KeyLabelColemak';
+import { KeyLabelCroatian } from './KeyLabelCroatian';
+import { KeyLabelCzech } from './KeyLabelCzech';
+import { KeyLabelDanish } from './KeyLabelDanish';
+import { KeyLabelDvorak } from './KeyLabelDvorak';
+import { KeyLabelDvorak_fr } from './KeyLabelDvorak_fr';
+import { KeyLabelDvp } from './KeyLabelDvp';
+import { KeyLabelEstonian } from './KeyLabelEstonian';
+import { KeyLabelFinnish } from './KeyLabelFinnish';
+import { KeyLabelFrench } from './KeyLabelFrench';
+import { KeyLabelFrench_afnor } from './KeyLabelFrench_afnor';
+import { KeyLabelFrench_osx } from './KeyLabelFrench_osx';
+import { KeyLabelFr_ch } from './KeyLabelFr_ch';
+import { KeyLabelGerman } from './KeyLabelGerman';
+import { KeyLabelGerman_ch } from './KeyLabelGerman_ch';
+import { KeyLabelGerman_osx } from './KeyLabelGerman_osx';
+import { KeyLabelGreek } from './KeyLabelGreek';
+import { KeyLabelHebrew } from './KeyLabelHebrew';
+import { KeyLabelHungarian } from './KeyLabelHungarian';
+import { KeyLabelIcelandic } from './KeyLabelIcelandic';
+import { KeyLabelIrish } from './KeyLabelIrish';
+import { KeyLabelItalian } from './KeyLabelItalian';
+import { KeyLabelItalian_osx_ansi } from './KeyLabelItalian_osx_ansi';
+import { KeyLabelItalian_osx_iso } from './KeyLabelItalian_osx_iso';
+import { KeyLabelJP } from './KeyLabelJP';
```

Comment:
@Silverquark The compile error occurred at this line. The correct file name is `./KeyLabelJp`. Could you modify this?